### ISSUE TITLE
Are there Environment Variables for the default administrative username and password?

### ISSUE THREAD

**Issue by KLittle1996** [louislam/uptime-kuma#1185]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

My plan is to run a number UK containers to serve status pages for each respective department in my organisation as UK does not yet support the feature. Plan is to build the container and deploy to fargate or ECS, most likely fargate.

One thing i'd like to do is standardize the default administrative user, so that if i need to nuke and redeploy or simply deploy new containers that i wont need to worry about setting it up. In a perfect world, i'd build the username and password in as GH secrets for an action to build the container and then push it over to ECR. That's a lot trickier to do if i can't define those variables in the compose file.

So in short, are there environment variables available to use to define the username and password for the account on UK upon first run-time of the container?

### 🐻 Uptime-Kuma Version

1.11.3

### 💻 Operating System and Arch

debian on WSL-2

### 🌐 Browser

Edge 97.0.1072.55

**Comment by louislam** [comment]
No such variables, but you can reset the password. Please check wiki for more details.

**Comment by KLittle1996** [comment]
> No such variables, but you can reset the password. Please check wiki for more details.

Is this something that could be added into the development pipeline as a feature reqest?

**Comment by zeGzD** [comment]
> No such variables, but you can reset the password. Please check wiki for more details.

I guess the point here was to be able to provision the default username/password when deploying the application.
In this way no need to be prompted to set it on first login which can be very convenient.

**Comment by ToshY** [comment]
@louislam Is this something that you're open to add if someone provided a PR?

**Comment by zeGzD** [comment]
It could probably be useful to open this issue again, isn't it?
